### PR TITLE
chore(circleci): add automated releases on merge with master

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by elnyry-sam-k** [comment]
LGTM, @mdebarros can you please confirm?

**Review by elnyry-sam-k — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -200,6 +203,71 @@ jobs:
...
+      - checkout
+      - run:
+          name: Fetch updated release branch
+          command: |
+            git config user.email ${GIT_CI_EMAIL}
+            git config user.name ${GIT_CI_USER}
+            git fetch origin
+            git checkout origin/${CIRCLE_BRANCH}
+      - run:
+          # Note: this is rather imperfect, but will do for now
```
Comment (lewisdaly) [inline .circleci/config.yml:251]: I'd love to find a better fix to this at some stage

Comment (kleyow) [reply .circleci/config.yml:251]: I'll make a ticket.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**